gui/src/lib.rs

```diff
@@ -14,8 +14,19 @@ pub mod utils;
 
 use liana::Version;
 
-pub const VERSION: Version = Version {
+pub const RETAILER_NAME: &str = "21st Capital";
+
+#[derive(Debug, Clone)]
+pub struct RetailerVersion(Version);
+
+impl std::fmt::Display for RetailerVersion {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        write!(f, "{} - {}", self.0, RETAILER_NAME)
+    }
+}
+
+pub const VERSION: RetailerVersion = RetailerVersion(Version {
```

Comment:
I am afraid that each change of the Version in upstream will conflict with this commit.
I think it would be better to have VERSION and RETAILER_VERSION = RetailerVersion(VERSION)

Reply:
and use RETAILER_VERSION in settings

Reply:
That's a good point thanks. I've now updated this :relieved: